### ISSUE TITLE
No feedback on successful authentication

### ISSUE THREAD

**Issue by mtmkls** [sddm/sddm#646]
The login window just freezes for about 10 seconds (until the KDE splash screen is loaded). I'm using the default Breeze theme.

**Comment by mtmkls** [comment]
Yes, the greeter's job is to quit when we've logged in. It doesn't do that. It stays frozen for several seconds, and I don't know if the password was accepted or not.

**Comment by davidedmundson** [comment]
It does. Just X continues rendering the last thing in the buffer.

**Comment by mtmkls** [comment]
Software usability 101: there must be an immediate feedback to user action. Freezing for several seconds is not a feedback.

**Comment by plfiorini** [comment]
If I recall how this works (it's been a while since I worked on SDDM), we close the greeter process as soon as the password is accepted and the user session is launched. This means that Xorg just shows the last buffer.

I think we can do better: never close the greeter process would allow themes to show something appropriate, then switch vt when the user session process is ready. When the user session process quits we just switch the vt back to the greeter.

The downside is that we no longer save the greeter process memory, although we can try to minimize it by hiding the greeter windows.
